.github/workflows/integration-sandbox.yml

```diff
@@ -10,8 +10,28 @@ jobs:
   main:
     name: Build/test integration sandbox
     runs-on: ubuntu-latest
-
+    services:
+      postgres:
+        image: ghcr.io/latticexyz/postgres-wal-logical:latest
```

Comment:
need a custom postgres image here that adds this to the postgres config:
```
wal_level = logical
max_replication_slots = 1
max_wal_senders = 1
```

postgres needs to restart after changing these config params, so the only way i found to make it work in the CI was to create a new docker image that has this config from the start